common/app/views/fragments/analytics/google.scala.html

```diff
@@ -30,8 +30,8 @@
     }
 
     function getAnalyticsTitle() {
-        var analyticsTitle = (guardian.config.page.webTitle || '').toLowerCase();
-        return (analyticsTitle === 'network front') ? getAnalyticsEdition() + ' ' + analyticsTitle : analyticsTitle;
+        var analyticsTitle = (guardian.config.page.webTitle || '');
+        return (analyticsTitle === 'Network Front') ? getAnalyticsEdition() + ' network front' : analyticsTitle;
```

Comment:
The title in apps is "uk network front", so this needs to be lowercased here. It's unfortunate that apps and web's backends don't produce similar titles for these things, the algorithm here ends up being quite unnatural.